.github/CONTRIBUTING.md

```diff
@@ -4,12 +4,24 @@ These are the guidelines for contributing to this repository.
 
 ## Issues
 
-File an issue if you think you've found a bug. Please describe
+Please follow the each template by the type of your issue.
 
-1. How can it be reproduced
-2. What was expected
-3. What actually occurred
-4. What version of the involved component
+- [Bug Report](https://github.com/naokazuterada/MarkdownTOC/issues/new)
+- [Feature Request](https://github.com/naokazuterada/MarkdownTOC/issues/new?template=feature.md)
+- [Question](https://github.com/naokazuterada/MarkdownTOC/issues/new?template=question.md)
+
+### Way of operation
+
+1. The contributors will `Close` these Issues or PR
```

Comment:
Do you mean maintainers?

Suggestion: "The maintainers will `Close` the issue or PR"

Reply:
Thanks you are right.